sabaki/scripts/docker-katago-raw.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd ${SCRIPT_DIR}
+
+GIT_ROOT=$(git rev-parse --show-toplevel)
+cd ${GIT_ROOT}
+
+docker run \
+    -v sabaki/models:/models \
```

Comment:
does this not need a `--gpus device=0`?

looks like `docker-katago-raw.sh` and `docker-katago-custom.sh` could be combined into one file for cleanliness if you want to be diligent — the only difference is calling  `/engines/KataGo-custom/cpp/katago` vs. `/engines/KataGo-raw/cpp/katago`

Reply:
Good suggestion, have combined the two scripts.